### ISSUE TITLE
service-catalog installer should use the same template used by oc cluster up

### ISSUE THREAD

**Issue by pmorie** [openshift/openshift-ansible#5039]
The installer and `oc cluster up` currently use different templates.  We should deduplicate to the greatest possible degree, preferably making the installer use the exact same templates as `oc cluster up`.

**Comment by pmorie** [comment]
@jpeeler this one is for you.

**Comment by jpeeler** [comment]
I've looked into this, but I'm unsure what the best approach is for keeping all the files in sync. Both sides are using templates, ansible with jinja and origin with the openshift format (oc process). I believe that oc cluster up should be copying what ansible is doing since that will be the method customers use to install. Does a script need to be made to verify or should a comment be left in origin where to look in openshift-ansible? Note that during the v0.0.14 service catalog rebase I mirrored the RBAC rules nearly identically in origin to what's present here.
